lib/msal-browser/src/interaction_client/StandardInteractionClient.ts

```diff
@@ -211,22 +211,32 @@ export abstract class StandardInteractionClient extends BaseInteractionClient {
      * @param requestAuthority
      * @param requestCorrelationId
      */
-    protected async getDiscoveredAuthority(requestAuthority?: string): Promise<Authority> {
+    protected async getDiscoveredAuthority(requestAuthority?: string, requestAzureCloudOptions?: AzureCloudOptions): Promise<Authority> {
         this.logger.verbose("getDiscoveredAuthority called");
         const authorityOptions: AuthorityOptions = {
             protocolMode: this.config.auth.protocolMode,
             knownAuthorities: this.config.auth.knownAuthorities,
             cloudDiscoveryMetadata: this.config.auth.cloudDiscoveryMetadata,
-            authorityMetadata: this.config.auth.authorityMetadata
+            authorityMetadata: this.config.auth.authorityMetadata,
         };
 
-        if (requestAuthority) {
+        // Fetch the authority from request if provided, default if the user authority is not defined
+        const userRequestedAuthority = requestAuthority ? requestAuthority : Constants.DEFAULT_AUTHORITY;
```

Comment:
This should fallback to the config authority (which will be the default authority if that was not provided either)
```suggestion
        const userRequestedAuthority = requestAuthority ? requestAuthority : this.config.auth.authority;
```

Reply:
Edited offline and pushed